.vscode/launch.json

```diff
@@ -0,0 +1,43 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Launch Package",
+      "type": "go",
+      "request": "launch",
+      "mode": "debug",
+      "program": "${workspaceRoot}/src",
+      "args": ["-config=${workspaceRoot}/themes/jandedobbeleer.omp.json"]
+    },
+    {
+      "name": "Launch tests",
+      "type": "go",
+      "request": "launch",
+      "mode": "test",
+      "program": "${workspaceRoot}/src",
+      "args": ["-test.v"]
+    },
+    {
+      "name": "Print debug",
+      "type": "go",
+      "request": "launch",
+      "mode": "debug",
+      "program": "${workspaceRoot}/src",
+      "args": [
+        "--debug",
+        "-config=${workspaceRoot}/themes/jandedobbeleer.omp.json"
+      ]
+    },
+    {
+      "name": "Print init pwsh",
+      "type": "go",
+      "request": "launch",
+      "mode": "debug",
+      "program": "${workspaceRoot}/src",
+      "args": ["--print-init","-shell=pwsh","-config=${workspaceRoot}/themes/jandedobbeleer.omp.json"]
```

Comment:
can you make the args consistent: `--shell`, same for the commands above. 🙏🏻

Reply:
Which syntax do you prefer single or double dash?
`-config=xxx` vs `-config xxx`
The second one is not really pretty in launch.json:
```
 {
      "name": "Print init pwsh",
      "type": "go",
      "request": "launch",
      "mode": "debug",
      "program": "${workspaceRoot}/src",
      "args": [
        "-print-init",
        "-shell=pwsh",
        "-config=${workspaceRoot}/themes/jandedobbeleer.omp.json"
      ]
}
```
vs
```
{
      "name": "Print init pwsh",
      "type": "go",
      "request": "launch",
      "mode": "debug",
      "program": "${workspaceRoot}/src",
      "args": [
        "-print-init",
        "-shell",
        "pwsh",
        "-config",
        "${workspaceRoot}/themes/jandedobbeleer.omp.json"
      ]
}
```

Reply:
`--config=config`